### PR TITLE
Include new vivo icons

### PR DESCRIPTION
**Pull request by tuentisre**
New version of icons exported from [Vivo](https://www.figma.com/file/IrcHGIgsF5Cq4ZX1LRhuis/Vivo-New-(Beta)?type=design&node-id=2625-199) 
 - [x] [Review readme import-figma-icons](https://github.com/Telefonica/mistica-icons/tree/import-figma-icons)

### CODE REVIEW

**Comment by yceballost** [comment]
Please @aweell double check icon names. I would like to avoid future breaking changes

File: icons/vivo-new/regular/care-smartphone-regular.svg
Comment (yceballost) [inline icons/vivo-new/regular/care-smartphone-regular.svg:1]: smartphone is not a "common" usage for this concept in current icons
image

donno if this is important or not

File: icons/vivo-new/regular/lightbulb-regular.svg
Comment (yceballost) [inline icons/vivo-new/regular/lightbulb-regular.svg:1]: lightbulb or light-bulb?

Comment (yceballost) [reply icons/vivo-new/regular/lightbulb-regular.svg:1]: dont care, there are already icons with this name

File: icons/vivo-new/regular/care-smartphone-regular.svg
Comment (aweell) [reply icons/vivo-new/regular/care-smartphone-regular.svg:1]: I would use `care-mobile-regular`

File: icons/vivo-new/light/random-light.svg
Comment (aweell) [inline icons/vivo-new/light/random-light.svg:1]: Maybe use `shuffle` instead of `random`?

Comment (yceballost) [reply icons/vivo-new/light/random-light.svg:1]: yes, better

File: icons/vivo-new/light/amplifier-signal-light.svg
Comment (aweell) [inline icons/vivo-new/light/amplifier-signal-light.svg:1]: Me pregunto si relacionado con `IconRepeaterRegular`, caen los dos en el mismo saco y deberían ser:

* `SignalRepeater`
* `SignalAmplifier`

Comment (yceballost) [reply icons/vivo-new/light/amplifier-signal-light.svg:1]: Existe ya un icono IconRepeaterRegular en mistica-icons debería usar el mismo nombre para la equivalencia?

File: icons/vivo-new/light/random-light.svg
Comment (yceballost) [reply icons/vivo-new/light/random-light.svg:1]: cambiado

File: icons/vivo-new/regular/care-smartphone-regular.svg
Comment (yceballost) [reply icons/vivo-new/regular/care-smartphone-regular.svg:1]: changed

**Review by aweell — APPROVED** [review]

**Pull request merged**